code-injection/row_004_Adelabumowe_typescript_vuln.ts

```
@@ -0,0 +1,77 @@
+});
+
+// Insecure deserialization
+app.post('/deserialize', (req, res) => {
+    const data = req.body.data;
+    const obj = JSON.parse(data);
+    res.send(obj);
+});
+
+// Using deprecated function
+const unsafeEval = (code: string) => {
+    eval(code);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)